src/ray/common/ray_config_def.h

```diff
@@ -318,3 +318,7 @@ RAY_CONFIG(bool, gcs_actor_service_enabled,
 
 /// Whether start the Plasma Store as a Raylet thread.
 RAY_CONFIG(bool, put_small_object_in_memory_store, false)
+
+RAY_CONFIG(bool, gcs_placement_group_service_enabled,
```

Comment:
I don't think a flag is needed here, since it has no effect unless the user actually tries using the API.